applicationinsights-react-js/src/ReactPlugin.ts

```diff
@@ -33,12 +36,22 @@ export default class ReactPlugin extends BaseTelemetryPlugin {
         let _unlisten: any;
         let _pageViewTimer: any;
         let _pageViewTracked:boolean;
+        let properties: PropertiesPlugin;
+
 
         dynamicProto(ReactPlugin, this, (_self, _base) => {
             _initDefaults();
 
             _self.initialize = (config: IConfiguration & IConfig, core: IAppInsightsCore, extensions: IPlugin[], pluginChain?:ITelemetryPluginChain) => {
                 super.initialize(config, core, extensions, pluginChain);
+                
+                let thePlugin = core.getPlugin<PropertiesPlugin>(PropertiesPluginIdentifier);
+                if (thePlugin) {
+                    properties = thePlugin.plugin;
+                }
+                objDefine(_self, "context", {
+                    g: () => properties.context
```

Comment:
There are actually some ( rare ) cases where the Properties plugin will not be present, so we should handle that here as well, as the goal of the SDK is that we should not throw any unexpected exceptions

Reply:
I think returning null (or undefined) when there is no properties would work.

Also as we are targeting es5, to aid within minification we are not using the `?.` operator as it generally generates more javascript than what we can do manually.

So please use
```ts
() => properties ? properties.context : null
```

rather than 
```ts
() => properties?.context || null
```

In the next major version (when we change the default targeted version of ES) we can and will change to using the 2nd option as that will produce the smaller code.